src/applications/personalization/profile/components/ProfileInfoCard.jsx

```diff
@@ -0,0 +1,213 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { numberBetween } from '../../common/proptypeValidators';
+
```

Comment:
For the classNames below. 

This is deprecating the previously used `prefixUtilityClasses` function. I found the old utility usage somewhat difficult to read, and just not ideal in my mind. The new composition of classNames uses one of the already well established packages [classNames](https://github.com/JedWatson/classnames) and assembles them with their full names, and I thought that was easier to read and manipulate.

I didn't remove the function from the platform code (although its only really used in one other place), but will evaluate it further and decide if that is something I want to do within the profile code after this launches.